web-app/react-app/components/ProfilePage/IntegrationKeyHandler/index.jsx

```diff
@@ -9,15 +9,20 @@ export default class IntegrationKeyHandler extends Component<{}> {
 
     render() {
         return (
-            <Panel header="Integration Keys">
-                <Row>
-                    <IntegrationKeyHandlerSegment
-                        service="ETHEREUM"
-                        name="Ethereum"
-                        inputFields={['privateKey']}
-                        tableFields={['address']}
-                    />
-                </Row>
+            <Panel>
+                <Panel.Heading>
```

Comment:
This affects Kare's branch, please give him a heads up (or update react-bootstrap in Kare's branch too)